src/Styles/homePage.css

```diff
@@ -0,0 +1,5 @@
+.homePage{
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+}
```

Comment:
Styled Components are also a great way to make unique selectors and good css for react, just to take a look: https://styled-components.com/